docs/DeveloperGuide.md

```diff
@@ -229,6 +232,9 @@ follows the sequence.
 The following sequence diagram shows how the delete policy operation works:
 ![DeletePolicySequenceDiagram0](images/DeletePolicySequenceDiagram.png)
 
+**Alternatives Considered**:
```

Comment:
For Alternatives, can we standardize is the following format:

#### Alternatives considered:

as this follows the format already in the DevGuide.

I will merge this PR and make the necessary changes later, just keep this in mind!